### ISSUE TITLE
Value.integerValue is type String yet Value.doubleValue is type double

### ISSUE THREAD

**Issue by jimmyff** [dart-lang/googleapis#91]
Value.integerValue is of type String however Value.doubleValue is of type double and Value.booleanValue is type bool. This is inconsistent.

**Comment by kevmoo** [comment]
Yup. This is because integer values can go beyond the range that can be supported safely with `int` – especially on the web.

I've pondered adding helpers in these cases so you can access the int value if it's valid.
